packages/wallet/src/wallet.ts

```diff
@@ -87,20 +87,20 @@ export default class Wallet extends AbstractWallet {
    * @param transactionRequestLike - TransactionRequestLike
    * @returns string - Signature a ECDSA 64 bytes
```

Comment:
The `@return` doc for the `signTransaction` method should contain a `Promise<>`:
 - `@returns Promise<string> - Signature a ECDSA 64 bytes`